beetmoverscript/data/release_beetmover_task_schema.json

```diff
@@ -46,9 +46,12 @@
                                 "items": {
                                     "type": "string"
                                 }
+                            },
+                            "zipExtract": {
+                                "type": "boolean"
                             }
                         },
-                        "required": ["taskId", "taskType", "paths", "locale"]
```

Comment:
Why trim `locale` here?

Reply:
geckoview tasks don't specify a locale to beetmove. That's why I removed it from the required fields. I'm fine putting it back and creating a different schema. What do you think?

Reply:
This is my fault from the very beginnings of beetmoverscript. In `shipitscript` and `bouncerscript` we have task schemas for each of the possible tasks and full integration tests as we should have! For beetmover unfortunately it is what it is. So to answer your question, ideally, we add a new schema so that we don't touch existing ones (like trimming the `locale` or `hashType` or `platform` to fit new maven tasks). Old beetmover jobs still have those, hence be tested against those correctly.

Again, this is not blocking so we can definitely follow-up in a separate PR. I have a backlog task to take over (hopefully) within declarative artifacts, to rewrite all the tasks to have integration tests for all possible beetmover jobs so this work will be undertaken anyway at some point this quarter so not sure it's worth time investing now. If it's easy for you to tweak tasks to fit to a new schema and not touch existing, that's be ideal, but again, we're going to do that sooner or later anyway for all tasks. Up to you, am fine either way 👍

Reply:
Got! I'd prefer to make it in a follow up. I think we're safe for now because beetmoverscript relies on `locale` being defined. So if a task def misses one, a `KeyError` will be raised somewhere

Reply:
👍